src/internal/components/selectable-item/index.tsx

```diff
@@ -22,6 +22,7 @@ export type SelectableItemProps = BaseComponentProps & {
   ariaPosinset?: number;
   ariaSetsize?: number;
   highlightType?: HighlightType['type'];
+  'aria-describedby'?: string;
```

Comment:
There is a type safety issue: https://github.com/microsoft/TypeScript/issues/32447

`aria-` and `data-` attributes are not type checked. This is why we use camel case names for all other aria props

Sorry for not spotting it earlier, but I think you want to address this issue

Reply:
updated